### PR TITLE
feat: adding hero section

### LINKED ISSUE #2: Hero section (HTML + CSS)

### PR DESCRIPTION
**Pull request by mateuszkierylo**

### CODE REVIEW

**Review by edytaedyta — APPROVED** [review]
ok

File: src/css/about.css [hunk 1]
```diff
@@ -39,7 +38,7 @@
     font-size: 14px;
     line-height: 1.714;
     text-align: center;
-    color:  #12141780;
+    color: #12141780;
```
Comment (PiotrJasinski1995) [inline src/css/about.css:41]: it should be variable

File: src/css/header.css [hunk 2]
```diff
@@ -77,21 +77,14 @@
...
-        outline: 1px solid #1214171A;
-        outline-offset: 8px;
-        border-radius: 2px;
-    }
     
     .header-icon:hover,
     .header-icon:active {
         fill: var(--green-color);
-        outline-color: var(--green-color);
+        outline-color:#4DC2741A;
```
Comment (PiotrJasinski1995) [inline src/css/header.css:87]: it should be variable

File: src/css/hero.css [hunk 3]
```diff
@@ -0,0 +1,201 @@
...
+    display: none;
+}
+
+.mobile-hero-image {
+    display: block;
+    border-radius: 20px;
+    overflow: hidden;
+}
+
+.hero-tittle {
```
Comment (PiotrJasinski1995) [inline src/css/hero.css:27]: "title", not "tittle"

File: src/css/hero.css [hunk 4]
```diff
@@ -0,0 +1,201 @@
...
+    text-align: left;
+    position: absolute;
```
Comment (PiotrJasinski1995) [inline src/css/hero.css:29]: position absolute is not a good idea, better use some padding

File: src/partials/about.html [hunk 5]
```diff
@@ -44,6 +43,11 @@ <h3 class="about-list-header">04</h3>
       </ul>
       <div class="about-picture">
         <picture>
+          <source srcset="
```
Comment (PiotrJasinski1995) [inline src/partials/about.html:46]: it should be just  with src and source set, look at "proposal section"

File: src/partials/about.html [hunk 6]
```diff
@@ -44,6 +43,11 @@ <h3 class="about-list-header">04</h3>
...
+              ../img/memoji.png 1x,
+              ../img/memoji@2x.png 2x
+            " />
+          <img class="about-picture-img" src="../img/memoji.png" alt="Smiling girl pointing her finger" />
```
Comment (PiotrJasinski1995) [inline src/partials/about.html:50]: wrong path, check where the index file is located relative to the images, so "./", not "../"

File: src/partials/hero.html [hunk 7]
```diff
@@ -1,3 +1,28 @@
...
+  <div class="container hero-content">
+      <div class="hero-tittle">
+        <h1>Discover the joy of learning <span class="english-green">English</span> with us!</h1>
+      </div>
+      <button class="btn cta-button">Leave an application</button>
+      
+      <div class="round-button"></div>
+      
+      <div class="desktop-hero-image">
+        <img src="../img/smiley-girl-wearing-headphones-desktop.jpg"
```
Comment (PiotrJasinski1995) [inline src/partials/hero.html:11]: wrong, the image is background-image, not , you should put this in the css!
and you should put also the gradient from Figma project, I don't see this

File: src/partials/hero.html [hunk 8]
```diff
@@ -1,3 +1,28 @@
...
+      <div class="tablet-hero-image">
+        <img src="../img/smiley-girl-wearing-headphones-tablet.jpg"
+          srcset="../img/smiley-girl-wearing-headphones-tablet.jpg 1x, ../img/smiley-girl-wearing-headphones-tablet@2x.jpg 2x"
+          alt="Girl studying with headphones">
+      </div>
+      <div class="mobile-hero-image">
+        <img src="../img/smiley-girl-wearing-headphones-mobile.jpg"
+          srcset="../img/smiley-girl-wearing-headphones-mobile.jpg 1x, ../img/smiley-girl-wearing-headphones-mobile@2x.jpg 2x"
+          alt="Girl studying with headphones">
+      </div>
```
Comment (PiotrJasinski1995) [inline src/partials/hero.html:24]: as I said before, everything with  should be removed there

File: src/public/icons.svg [hunk 9]
```diff
@@ -15,5 +15,8 @@
 <path stroke-linejoin="miter" stroke-linecap="round" stroke-miterlimit="4" stroke-width="3.4286" d="M8.671 24.468h35.329"></path>
 <path stroke-linejoin="miter" stroke-linecap="round" stroke-miterlimit="4" stroke-width="3.4286" d="M19.602 14.771h24.398"></path>
 </symbol>
+<symbol id="icon-arrow" viewBox="0 0 32 32">
```
Comment (PiotrJasinski1995) [inline src/public/icons.svg:18]: why add new icon, the icon "arrow-down-icon" was already there

File: src/partials/hero.html [hunk 10]
```diff
@@ -1,3 +1,28 @@
 <section id="hero" class="section hero-section">
-  <div class="container">Hero</div>
+  <div class="container hero-content">
+      <div class="hero-tittle">
+        <h1>Discover the joy of learning <span class="english-green">English</span> with us!</h1>
+      </div>
+      <button class="btn cta-button">Leave an application</button>
+      
+      <div class="round-button"></div>
```
Comment (PiotrJasinski1995) [inline src/partials/hero.html:8]: 1. that should not be <div> with background image! this is an svg.
2. if position absolute here, it is better to do from the botton and right, not top left

File: src/img/arrow-2.png
Comment (PiotrJasinski1995) [inline src/img/arrow-2.png:1]: what is the reason to add this img, arrow is the svg icon

File: src/img/arrow.png
Comment (PiotrJasinski1995) [inline src/img/arrow.png:1]: what is the reason to add this img, arrow is the svg icon

**Review by PiotrJasinski1995 — COMMENTED** [review]
these errors should be fixed

**Pull request merged**